check_pgactivity

```diff
@@ -7241,15 +7248,10 @@ SELECT
             elsif ($sequences{$seq}->{TYPE} eq 'int8') {
                 $max_value=9223372036854775807;
             }
-            else
-            {
-                # We're not going to try to guess. This is not a serial, trust
-                # the dba/developer
-                delete $sequences{$seq};
-                next;
-            }
+	    # When we can't link the sequence to a column, we true the dba/developer
```

Comment:
s/true/trust/

Reply:
done in 73caa656410f68e02c2dcbc4499b3bd3add77fa2